### ISSUE TITLE
3.4.4rc1 fails to install

### ISSUE THREAD

**Issue by foxx** [yyuu/pyenv#515]
The following fails;

```
$ pyenv install 3.4.4rc1
Downloading Python-3.4.4rc1.tgz...
-> https://www.python.org/ftp/python/3.4.4/Python-3.4.4rc1.tgz
/home/vagrant/.pyenv/plugins/python-build/bin/python-build: line 223: pushd: Python-3.4.4: No such file or directory

BUILD FAILED (Ubuntu 14.04 using python-build 20151222-7-g4c06f23)

Inspect or clean up the working tree at /tmp/python-build.20151231160648.28021
Results logged to /tmp/python-build.20151231160648.28021.log

Last 10 log lines:
/tmp/python-build.20151231160648.28021 ~
```

Looks like it's not detecting `rc1` on the end of the filename, instead going for `Python-3.4.4`.

**Comment by yyuu** [comment]
CPython 3.4.4 has already been released and 3.4.4rc1 is obsolete now. I'll just remove it instead of fixing. Thanks for reporting anyway

https://www.python.org/download/releases/3.4.4/

**Comment by foxx** [comment]
Ah great stuff, thank you!
